aod/rotki_assets.ipynb

```diff
@@ -0,0 +1,930 @@
tWsXn376KR999BHffvttsfOMRiMDBgxg27Zt/Pzzzxw9epR33nkHjaZ46eVz587RvXt3WrduzdKlS3FwcCh2zunTp+nfvz/Dhg3j4MGDLFq0iK1btzJhwoRKPY/CwkKmT5/OgQMHWLlyJXFxcYwbN67Yea+88grvvPMOx44do23btpW6lyi/QYMGkZKSwpw5c9i5cyc7d+4EoKCggH/++YfRo0dz9913s2bNGvbt28frr79OQcF/G60FBATQu3dvFixYAMCCBQsYPXq06XhWVhaDBg1i//79Zh8nT56kR48eFnkO48aNY/DgwRZpqzz+/vtvBg0aRIMGDVCpVKxcubLG7i2ErZA+5xoWFBTExx9/jEqlolmzZhw6dIiPP/6YRx991Oy8P//8k127dnHs2DFT13FYWFix9mJiYujbty9Dhgzhk08+KbVc8//93/8xevRoU+9C06ZNmTVrFj179uTLL7/E0dGxQs/j4YcfNn0eFhbGrFmz6NSpE1lZWbi4uJiOvfXWW/Tt27dCbVdFSkoKU6ZMYcOGDcTHx+Pr68vgwYOZPn067u7uNRaHNSQnJxMTE8OcOXPo2rUrKpWK7du3m45v376dkJAQXn/9ddNjZ8+eLdbO6NGjeemllxg1ahRnzpzh/vvvNx3r0KEDy5YtIzQ01GaGrLKzs2nXrh0PP/wwQ4cOtXY4QtRJ0jNRw2677TazF/wuXbpw8uRJDAaD2Xn79++nUaNGNxyDzs3NpXv37gwdOpRPP/30hvs+HDhwgLlz55p1T/fr1w+j0UhsbGyFn0d0dDSDBg0iODgYV1dXevbsCWDWZQ7QsWPHCrddFRcvXuTixYt88MEHHD58mLlz57J+/XoeeeSRGo3jWvPmzcPb27vY8NPgwYMZM2YMAKtWraJDhw44OjoSFhbGtGnT0Ov1pnM/+ugj2rRpg7OzM0FBQTz11FNkZWWZjs+dO5ewsDBcXV0ZOHAg9vb2/PLLL/Tp0weA2bNnM3nyZM6cOcOjjz7K6dOnmTVrFitWrCgW79ChQ8nMzOTOO+8kKCiIBg0amI7df//9xMTEcOedd7J7924++OADmjZtip2dHf7+/sWGu0padbRy5cpSf1anTp3Kjz/+yKpVq1CpVKhUKqKioigoKGDChAkEBgbi6OhISEgI//d//1fm9z4yMpKIiAizxwoLC/Hx8WHevHkADBgwgBkzZjBkyJAy2xNClEySiVpKp9OVeY6DgwN33nkna9as4cKFCzc8Nysri8cff9ysa/rAgQOcPHnSbNnfta5dZrht2zZWrVrF6dOnyc7Opl+/fpw+fRpFUdDr9bi5uQGY5mCcO3cOoFhcVVlmaDQaadSoEV9++aXZ4/v27UOtVnP27Flat27NsmXLGDRoEE2aNKF37968/fbb/Prrr2YvzjVpxIgRGAwGVq9ebXrsypUr/Pbbbzz88MNs2bKFsWPHMnHiRI4ePcrXX3/N3Llzefvtt03nq9VqZs2axZEjR/jxxx/ZtGkTL730ktl9cnNzCQoKwsvLC61Wy//93//h6ekJQHh4OIcOHaJHjx58++23tG3blu3bt/PGG28Ui9fV1ZVBgwaRk5NDZmYm187R3rx5M4GBgXh5eXHXXXfx+uuvk5+fz5gxY2443FVeL7zwAiNHjqR///4kJCSQkJBA165dmTVrFqtXr2bx4sXExMQwf/5803yQGxk9ejS//vqrWeL1+++/k5OTI8mDEJakiBrTs2dPpWXLlmaPvfLKK0qLFi0URVGUkJAQ5eOPP1YURVGioqIUtVqtxMTElNjWDz/8oLi7uyt6vV4ZOXKk0rRpU+XChQum43/99ZcCKKmpqYqiKEpkZKTSp0+fCsW7dOlSZdmyZcrJkyeVW265RWncuLHSpk0bZdeuXQqgvPDCC8q2bduU2NhY5dlnn1UAZeLEiWb3f+GFF8zavOWWW5TJkycriqIoqampiq+vr/Lqq68qx44dU/bu3av07dtX6dWrV6kxvfDCC8rtt99u9tjzzz9f7LFrzZkzR/Hx8anQc7e0J598UhkwYIDp6w8//FAJCwtTjEaj0qdPH2XmzJlm5//0009KYGBgqe0tWbJE8fb2Nn39ww8/KICyf/9+s/NCQkKUBx54wPS10WhU/Pz8lC+//LLMmK9cuaJotVrl77//Nj3WpUsX5eWXXy71mt27dyuAkpmZaYrL3d3d7JwVK1Yo1/7pmTJlitKuXTvT1w8++KBy3333mV3zzDPPKL1791aMRmOZcV+rsLBQ8fHxUebNm2d6bNSoUUpERESJ5wPKihUrKnQPIYSiSM9EDYuPj+e5554jJiaGhQsXMnv2bCZOnFjsvJ49e9KjRw+GDRvGH3/8QWxsLOvWrWP9+vVm52k0GubPn0+7du3o3bs3ly5dKvG+L7/8Mtu3b2fChAmmCXOrVq264QTMYcOGMXToUMLDw3FxcaFv374cOnSI3Nxc7O3tUalUBAQEcPDgQX777TcANmzYYNbGsmXLTJ+fOHGC6Oho04S+zz77jPbt2zNz5kyaN29O+/bt+f777/nrr784ceJEiTGNHj2abdu2mYZTjEYjv/zyi9kkwWslJSUxffp0q9fvePTRR9mwYYOpp2bu3LmMGzcOlUrFgQMHeOutt8yGoB599FESEhLIyckBiubQ9OnTh4YNG+Lq6sqYMWNITk42HQewt7cvcZLrtY9d/T+7diiiNL6+vtx1112muhOxsbGmSZxXlXe4q6rGjRvH/v37adasGf/73/+K/ZyVRqvVMnLkSNNzyM7OZtWqVaX+vAghKkeSiRo2duxYcnNzufXWW3n66aeZOHFiqS90y5Yto1OnTowaNYqWLVvy0ksvFZtbAUV/MBcuXEirVq3o3bt3iS8Ubdu2ZfPmzZw4cYLu3bvTvn173nzzTbPx8Otdu8xw69at/PDDD0DRkMncuXOZO3cu4eHhDB8+3PTicX0yEx8fX+llhvPnzzc7tmXLFm6++WZatGhhWm2w
```

Comment:
This information gets continuously updated by rotki as the app runs. It has a periodic task that checks the chains and continuously pulls the latest data